manifests/test.pp

```diff
@@ -0,0 +1,7 @@
+class example::test {
```

Comment:
Only the name of the file seems a little arbitrary ?

Many `example::concat` or `example::tests`  (plural ) since presumably more than one in the future.

Really not very important though.